components/App/Navbar.vue

```diff
@@ -19,6 +19,16 @@ const { data, refresh } = useLazyFetch<any>(streamChangesEndpoint, {
 
 const isLive = computed(() => data.value?.title)
 
+const { addNotification } = useNotifications()
+
+watch(isLive,async(newValue,oldValue) => {
+  if(isLive==='live'){
+      addNotification({
+        heading: 'Heading',
+        body: 'Text'
+      })
+  }
+})
```

Comment:
This is the right way 👍

But a check to `if (newValue)` would be enough ☺️ 

```suggestion
watch(isLive,async(isLiveNow) => {
  if(isLiveNow){
      addNotification({
        heading: 'Heading',
        body: 'Text'
      })
  }
})
```

Also please format the code